docs/application/dotnet/guides/nui/xaml/TypeConverter.md

```diff
@@ -0,0 +1,66 @@
+# TypeConverters and XAML
```

Comment:
Hello, I am very sorry but I am not sure. I don't know what is the right way. I want to keep current form. I added links as you suggested. I don't think it will be a matter. I don't want to change.

Reply:
> Hello, I am very sorry but I am not sure. I don't know what is the right way. I want to keep current form. I added links as you suggested. I don't think it will be a matter. I don't want to change.

MS addresses the following terms about **DOCUMENTS**:
---
## Notice Specific to Documents Available on this Website
Certain documentation may be subject to explicit license terms separate from the terms contained here. To the extent the terms conflict, the explicit license terms control. Permission to use Documents (such as white papers, press releases, datasheets and FAQs) from the Services is granted, provided that (1) the below copyright notice appears in all copies and that both the copyright notice and this permission notice appear, (2) use of such Documents from the Services is for informational and non-commercial or personal use only and will not be copied or posted on any network computer or broadcast in any media, and (3) no modifications of any Documents are made. Accredited educational institutions, such as K-12, universities, private/public colleges, and state community colleges, may download and reproduce the Documents for distribution in the classroom. Distribution outside the classroom requires express written permission. Use for any other purpose is expressly prohibited by law, and may result in severe civil and criminal penalties. Violators will be prosecuted to the maximum extent possible.
---

I think we had better not to copy or modify any MS documents.

Reply:
@journey2w,

Thank you and I totally agree with you. We must not copy or modify any MS documents.